<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRange_Dialog</class>
 <widget class="QDialog" name="AddRange_Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>173</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Custom Channel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ChannelsSummary_GroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Channel in Dataset</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Count_Label">
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Count_LineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>367</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Selection_GroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="SelectionMin_Label">
        <property name="text">
         <string>Min.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SelectionMin_LineEdit">
        <property name="placeholderText">
         <string>15000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SelectionMax_Label">
        <property name="text">
         <string>Max.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SelectionMax_LineEdit">
        <property name="placeholderText">
         <string>20000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Selection_ColorButton">
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 240, 107);
border: 1px solid black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="AddRangeDialog_ButtonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Count_LineEdit</tabstop>
  <tabstop>SelectionMin_LineEdit</tabstop>
  <tabstop>SelectionMax_LineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>AddRangeDialog_ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddRange_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddRangeDialog_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRange_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
